From 6150318055f8f25dd75b52a1eab534245259e016 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Sun, 14 Jan 2024 17:03:14 +0000
Subject: [PATCH 2/4] no-bundled-apache-commons hard part

---
 .../SimpleXMLParserDocumentImpl.java          | 24 +------------------
 1 file changed, 1 insertion(+), 23 deletions(-)

diff --git a/core/src/com/biglybt/pifimpl/local/utils/xml/simpleparser/SimpleXMLParserDocumentImpl.java b/core/src/com/biglybt/pifimpl/local/utils/xml/simpleparser/SimpleXMLParserDocumentImpl.java
index 545e89083..28c1f2171 100644
--- a/core/src/com/biglybt/pifimpl/local/utils/xml/simpleparser/SimpleXMLParserDocumentImpl.java
+++ b/core/src/com/biglybt/pifimpl/local/utils/xml/simpleparser/SimpleXMLParserDocumentImpl.java
@@ -31,7 +31,6 @@ import java.util.Vector;
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 
-import org.apache.commons.lang.Entities;
 import org.w3c.dom.Document;
 import org.w3c.dom.Node;
 import org.xml.sax.*;
@@ -669,28 +668,7 @@ SimpleXMLParserDocumentImpl
 
 		     				String	replacement;
 
-		     				if ( 	ref.equals( "amp") 		||
-		     						ref.equals( "lt" ) 		||
-		     						ref.equals( "gt" ) 		||
-		     						ref.equals( "quot" )	||
-		     						ref.equals( "apos" ) 	||
-		     						ref.startsWith( "#" )){
-
-		     					replacement = new String( buffer, 0, buffer_pos );
-
-		     				}else{
-
-			     				int num = Entities.HTML40.entityValue( ref );
-
-		     					if ( num != -1 ){
-
-		     						replacement = "&#" + num + ";";
-
-		     					}else{
-
-		     						replacement = new String( buffer, 0, buffer_pos );
-		     					}
-		     				}
+							replacement = new String( buffer, 0, buffer_pos );
 
 		     				char[] chars = replacement.toCharArray();
 
-- 
2.43.0

