--- ./core/src/com/biglybt/platform/unix/startupScript	2025-03-02 20:53:39.874450174 +0000
+++ ./core/src/com/biglybt/platform/unix/startupScript	2025-04-12 14:17:53.274705195 +0100
@@ -36,6 +36,8 @@ JAVA_ARGS=$(grep -o '^[^#]*' ~/.biglybt/
 JAVA_VERSION=""
 isJ17=false
 JAVABIN="${JAVA_PROGRAM_DIR}java"
+JAVA_HEADER=$("$JAVABIN" -version 2>&1 | grep -i version | head -n 1)
+JAVA_VERSION=$(echo "${JAVA_HEADER}" | sed "s/^.* version \"\(.*\)\".*$/\1/")
 
 look_for_java()
 {
